session/src/main/scala/akka/stream/alpakka/cassandra/CqlSessionProvider.scala

```diff
@@ -79,4 +78,17 @@ object CqlSessionProvider {
       }
       .get
   }
+
+  /**
+   * The `Config` for the `datastax-java-driver`. The configuration path of the
+   * driver's configuration can be defined with `datastax-java-driver-config` property in the
+   * given `config`. `datastax-java-driver` configuration section is also used as fallback.
+   */
+  def driverConfig(system: ActorSystem, config: Config): Config = {
+    val driverConfigPath = config.getString("datastax-java-driver-config")
+    system.settings.config
+      .getConfig(driverConfigPath)
+      .withFallback(if (driverConfigPath == "datastax-java-driver") ConfigFactory.empty()
+      else system.settings.config.getConfig("datastax-java-driver"))
+  }
```

Comment:
Shouldn't this be part of `DefaultSessionProvider`?

Reply:
as commented above, the intention is that this should be the way to "find" the driver config for "all" custom session providers